### ISSUE TITLE
爱可可微博热门分享(3.26)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#111]
No 1. <a href="https://weibo.com/1402400261/O6KTU4o1u">浓缩时间轴：AI简史</a>
No 2. <a href="https://weibo.com/1402400261/O6MDExdGc">A Survey of LLM Surveys：精心分类的LLM综述大列表</a>
No 3. <a href="https://weibo.com/1402400261/O6PqzpRsK">#免费##抽奖# 携手@博文视点Broadview 送出3本《码农翻身2》</a>
No 4. <a href="https://weibo.com/1402400261/O6Kvn0zQo">Grok-1迎来PyTorch+HuggingFace优化版：推理加速3.8倍,开发门槛大幅降低</a>
No 5. <a href="https://weibo.com/1402400261/O6MEEpoLM">Sycamore：用于复杂非结构数据(文档、幻灯片、音频等)的开源项目，提供基于LLM的搜索和分析等服务</a>
No 6. <a href="https://weibo.com/1402400261/O6KzqeZW5">LeaderboardFinder：找到适合特定场景的大模型排行榜</a>
No 7. <a href="https://weibo.com/1402400261/O6KsWB6nm">从技术到创意：Sora如何重塑艺术家的工作流程</a>
No 8. <a href="https://weibo.com/1402400261/O6Mk5pPi0">用LLM在《街头霸王III》中实时战斗</a>
No 9. <a href="https://weibo.com/1402400261/O6MzDlMsl">'rag-search - RAG Search API' GitHub: github.com/thinkany-ai/rag-search ......</a>
No 10. <a href="https://weibo.com/1402400261/O6MLs9Z8n">具身智能相关资源大列表</a>
No 11. <a href="https://weibo.com/1402400261/O6MIvy3Ea">Prompt Quill：全球第一个基于RAG的提示工程助手，基于320万个可用提示库，目的是为了帮用户更好地创建用于生成图像的提示</a>
No 12. <a href="https://weibo.com/1402400261/O6su8CHHx">Nvidia提供的在线免费新课</a>
No 13. <a href="https://weibo.com/1402400261/O6Kxo2nsI">Marigold-LCM：无监督单目深度估计模型Demo，可以从单张图像中高效预测各像素点的深度信息，不需要额外的监督信号如深度图或者立体图像对</a>
No 14. <a href="https://weibo.com/1402400261/O6KqpyYCe">#免费##抽奖# 携手@图灵文化 @图灵新知 送出3本《hello 算法》</a>
No 15. <a href="https://weibo.com/1402400261/O6D41EcNQ">Cookbook to Craft Good Code：一个帮助编写高质量代码的开源指南，强调代码的可读性、简洁性和易维护性</a>
No 16. <a href="https://weibo.com/1402400261/O6MAzxj4M">AIOS: 基于大型语言模型的操作系统，允许操作系统像“具有灵魂”一样运行，AIOS 是一个旨在优化资源分配、促进跨agent上下文切换、实现agent并发执行和提供LLM agent工具服务</a>
No 17. <a href="https://weibo.com/1402400261/O6Ml4jgun">Friend: 开源AI可穿戴设备，可实时分析语音并提供主动反馈和建议</a>
No 18. <a href="https://weibo.com/1402400261/O6MP72Yu5">神经编程智能相关资源大列表</a>
No 19. <a href="https://weibo.com/1402400261/O6Dc9bvae">NineRec: 多模态大规模数据集和多领域推荐系统基准</a>
No 20. <a href="https://weibo.com/1402400261/O6JDwwbT9">[RO] Learning Quadruped Locomotion Using Differentiable Simulation ......</a>
No 21. <a href="https://weibo.com/1402400261/O6MrcB2P7">'TinyVC : Towards AI voice conversion on CPU - a lightweight voice ......</a>
No 22. <a href="https://weibo.com/1402400261/O6MqojOyI">Tracecat：开源自动化平台，Tines / Splunk SOAR的开源替代</a>
No 23. <a href="https://weibo.com/1402400261/O6DdU75Wt">'Char detection base on crnn 字符（单字）检测基于CRNN' GitHub: github.com/fanqie03/char-detection ......</a>
No 24. <a href="https://weibo.com/1402400261/O6D6NezKY">Chat with MLX：高性能的macOS应用，将本地文档与一个个性化大型语言模型(LLM)连接起来。利用检索增强生成 (RAG)，可有效地搜索和查询文档</a>
No 25. <a href="https://weibo.com/1402400261/O5FCWiOLD">[LG] Understanding the Double Descent Phenomenon in Deep Learning ......</a>
No 26. <a href="https://weibo.com/1402400261/O6JsQC13B">[CL] The opportunities and risks of large language models in ......</a>
No 27. <a href="https://weibo.com/1402400261/O6GKGaUaB">晚安～ [月亮] ​</a>
No 28. <a href="https://weibo.com/1402400261/O6MJMfNJG">文生视频相关文献资源列表</a>
No 29. <a href="https://weibo.com/1402400261/O6Jpmj2sv">[CL]《CoLLEGe: Concept Embedding Generation for Large Language Models》</a>
No 30. <a href="https://weibo.com/1402400261/O6MPm0zl4">几篇论文实现代码</a>
No 31. <a href="https://weibo.com/1402400261/O6JiwhbBt">早！[太阳] ​</a>
No 32. <a href="https://weibo.com/1402400261/O6MKv317f">The Pipe：一个多模态工具，用于将现实世界的信息输入大语言模型，基于多核设计，通过精心设计的启发式方法，从文件、文件夹、网页等来源创建有意义的文本和图像提示</a>
No 33. <a href="https://weibo.com/1402400261/KaHAND2qK">Data Visualization Guide for Presentations, Reports, and Dashboards：面向演示、报告和面板的(Python)数据可视化指南</a>
No 34. <a href="https://weibo.com/1402400261/O6Dfbp8b8">Multimodal Gamer：在电脑上用多媒态模型打游戏的框架</a>
No 35. <a href="https://weibo.com/1402400261/O6Mg4CA8b">'skerch: Sketched matrix decompositions for PyTorch' GitHub: ......</a>
No 36. <a href="https://weibo.com/1402400261/O6MFB1yZY">'FlowJax: Distributions and Normalizing Flows in Jax' GitHub: ......</a>
No 37. <a href="https://weibo.com/1402400261/O6JJXjo0J">[LG] Accelerated Objective Gap and Gradient Norm Convergence ......</a>
No 38. <a href="https://weibo.com/1402400261/O6JjZDS9k">[CL]《LLM2LLM: Boosting LLMs with Novel Iterative Data Enhancement》</a>
No 39. <a href="https://weibo.com/1402400261/O6CJKnpvw">'LapisCV - 开箱即用的 Obsidian / Typora 简历' GitHub: github.com/BingyanStudio/LapisCV ......</a>
No 40. <a href="https://weibo.com/1402400261/O6AfZcveI">[CL] LlamaFactory: Unified Efficient Fine-Tuning of 100+ Language ......</a>
No 41. <a href="https://weibo.com/1402400261/O6tHorWH1">Lapdev：自托管远程开发环境管理系统，可以在服务器或云服务器上建立和管理远程开发环境</a>
No 42. <a href="https://weibo.com/1402400261/Ki075ekKh">Lumen：数据可视化分析框架</a>
No 43. <a href="https://weibo.com/1402400261/O6ML5kiCj">Vid2Persona：用 AI 描述视频角色的性格，并作为角色与您聊天</a>
No 44. <a href="https://weibo.com/1402400261/O6JjC8oyA">[LG]《Can large language models explore in-context?》</a>
No 45. <a href="https://weibo.com/1402400261/O6Jk5gU6i">LLM2LLM通过迭代地训练学生模型、评估识别错误样本、用教师模型生成新数据增强错误样本，实现了有效的基于语言模型的定向数据增强，显著提升了语言模型在低资源场景下的微调效果。//[CL]《LLM2LLM: Boosting LLMs with Novel Iterative Data Enhancement》</a>
No 46. <a href="https://weibo.com/1402400261/O6ilu3Wss">NLP入门指南：一文览尽自然语言处理的基本概念</a>
No 47. <a href="https://weibo.com/1402400261/O6CLFqb7N">Whispering：一个开源项目，旨在通过 OpenAI Whisper API 简化语音到文字转换体验</a>
No 48. <a href="https://weibo.com/1402400261/O6vvx4X9m">《告别焦虑：用行动让内心重获自由》</a>
No 49. <a href="https://weibo.com/1402400261/O6CK5oqfm">Gmeek：一个博客框架，使用 git 托管代码，简洁易用，无需本地部署，只需搭建和写作即可完成博客搭建。</a>
No 50. <a href="https://weibo.com/1402400261/O6JNAD6QM">今日推介(第1356期)：大型语言模型能在上下文中探索吗、用新型迭代数据增强来提升LLM、利用Spotlighting防御间接提示注入攻击、防止越狱把安全性定义清楚最重要、大型语言模型的概念嵌入生成 ......</a>
